jupyter_server/tests/selenium/test_multiselect.py

```diff
@@ -0,0 +1,65 @@
+def test_multiselect(notebook):
```

Comment:
Where are the other selenium tests - in particular the conftest.py file that setups up the 'notebook' instance.

I'm wondering if when the server was setup, the selenium directory was intentionally omitted and this file has "snuck in".

Reply:
You're right. I've been trying to catch these selenium tests and delete them. Looks like this one slipped through.

Reply:
Removing this directory now.